x/feeabs/ibc_module.go

```diff
@@ -201,7 +201,18 @@ func (am IBCModule) OnTimeoutPacket(
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) error {
-	// TODO: Resend request if timeout
-	// TODO: emit event
+	chancap := am.keeper.GetCapability(ctx, host.ChannelCapabilityPath(am.keeper.GetPort(ctx), am.keeper.GetChannelId(ctx)))
+	// Resend request if timeout
+	err := am.keeper.OnTimeoutPacket(ctx, chancap, packet) // If there is an error here we should still handle the timeout
+	if err != nil {
```

Comment:
We should add Logger here if func return an error. It will helpful for debug a lot